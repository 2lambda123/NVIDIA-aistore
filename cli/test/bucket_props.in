ais create bucket $BUCKET_1
ais set props $BUCKET_1 'mirror.enabled=true' 'mirror.copies=2'
ais set props $BUCKET_1 'aattrs=ro'

ais set props $BUCKET_1 'unknown.field' // FAIL "no value for"
ais set props $BUCKET_1 'unknown.field=true' // FAIL "unknown property"

ais show props $BUCKET_1
ais set props --reset $BUCKET_1

ais set props $BUCKET_1 --jsonspec '{"versioning": {"enabled": true, "validate_warm_get": true}}'
ais show props $BUCKET_1
