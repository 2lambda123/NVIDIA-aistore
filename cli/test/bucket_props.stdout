"$BUCKET_1" bucket created
Bucket props successfully updated
Bucket props successfully updated
PROPERTY	 VALUE
access		 GET,HEAD
checksum	 Type: xxhash | Validate: ColdGET
ec		 Disabled
lru		 Watermarks: 75%/90% | Do not evict time: 120m | OOS: 95%
mirror		 2 copies
provider	 ais
versioning	 Enabled | Validate on WarmGET: no
Bucket props successfully reset
Bucket props successfully updated
PROPERTY	 VALUE
access		 GET,PUT,DELETE,HEAD,ColdGET
checksum	 Type: xxhash | Validate: ColdGET
ec		 Disabled
lru		 Watermarks: 75%/90% | Do not evict time: 120m | OOS: 95%
mirror		 Disabled
provider	 ais
versioning	 Enabled | Validate on WarmGET: yes
PROPERTY		 VALUE
lru.capacity_upd_time	 10m
lru.dont_evict_time	 120m
lru.enabled		 true
lru.highwm		 90
lru.lowwm		 75
lru.out_of_space	 95
